### PR TITLE
Use i18n_helpers to provide i18n

### PR DESCRIPTION
**Pull request by rbino**
This currently only involves System Model descriptions. This PR migrates the data from the system_model_descriptions table to an embedded map in the system_model table and fixes the code and tests accordingly.

Signed-off-by: Riccardo Binetti <riccardo.binetti@secomind.com>

### CODE REVIEW

File: backend/priv/repo/migrations/20230111171104_embed_system_model_descriptions.exs [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    placeholders = %{timestamp: timestamp}
+
+    Repo.insert_all(SystemModel, upserts,
+      placeholders: placeholders,
+      conflict_target: [:id, :tenant_id],
+      on_conflict: {:replace, [:description, :updated_at]}
+    )
+
+    drop table("system_model_descriptions")
```
Comment (Pavinati) [inline backend/priv/repo/migrations/20230111171104_embed_system_model_descriptions.exs:80]: Are system_model_descriptions indexes dropped automatically?

**Review by Pavinati — CHANGES_REQUESTED** [review]
Remember to update the license headers and external license files with 2023

**Review by davidebriani — APPROVED** [review]

File: backend/test/edgehog_web/schema/query/system_model_test.exs [hunk 2]
```diff
@@ -143,10 +131,36 @@ defmodule EdgehogWeb.Schema.Query.SystemModelTest do
...
+
+      description = %{default_locale => "A system model", "it-IT" => "Un modello di sistema"}
+
+      %SystemModel{id: id} = system_model_fixture(hardware_type, description: description)
+
+      variables = %{id: Absinthe.Relay.Node.to_global_id(:system_model, id, EdgehogWeb.Schema)}
+
+      conn =
+        conn
+        |> put_req_header("accept-language", "fr-FR;q=0.9 it-IT;q=0.5")
```
Comment (szakhlypa) [inline backend/test/edgehog_web/schema/query/system_model_test.exs:157]: I think it should be comma here `fr-FR;q=0.9,it-IT;q=0.5`

File: backend/priv/repo/migrations/20230111171104_embed_system_model_descriptions.exs (hunk 1 above)
Comment (rbino) [reply backend/priv/repo/migrations/20230111171104_embed_system_model_descriptions.exs:80]: [Yes](https://www.postgresql.org/docs/current/sql-droptable.html)

**Review by Pavinati — APPROVED** [review]

**Review by szakhlypa — APPROVED** [review]

**Pull request merged**